README.md

```diff
@@ -1,2 +1,70 @@
 # browserify-i18n
 Browserify transform for internationalizing tagged strings
+
+
+## Install
+
+```
+$ npm install https://github.com/yola/browserify-i18n.git
+```
+
+## Usage
+
+For standard, non-optimized usage, do:
+
+```javascript
+var browserify = require('browserify');
+var i18n = require('browserify-i18n');
+var opts = {
+  locale: 'en',                         // The locale code to be used
+  localeDirs: ['./path/to/locale/dir/'] // Array of paths to locale .po files
```

Comment:
To illustrate which options must go to `i18n.fast` and which ones must go to the second argument of `b.transform`. We could use any option, but I think `global` is the only one you can give to a transform by default.